### PR TITLE
High latency doc

### PR DESCRIPTION
**Pull request by mattgwwalker**
Hi,

I added a note to the documentation of a Stream's desired latency indicating that 'high' may not be sufficient to obtain stable audio.

In my commit history there's a revert.  Does this impact you in any way?  Or do my commits just get merged into one at your end?

Cheers,

Matthew

### CODE REVIEW

**Comment by mattgwwalker** [comment]
Oh, for reference, we discussed this in #262.

File: sounddevice.py [hunk 1]
```diff
@@ -1643,6 +1643,12 @@ def __init__(self, samplerate=None, blocksize=None,
             next practical value -- i.e. to provide an equal or higher
             latency  wherever possible.  Actual latency values for an
             open stream may be retrieved using the `latency` attribute.
+
+            .. note:: Specifying the desired latency as 'high' does
+                not guarantee a stable audio stream. For reference, by
```
Comment (mgeier) [inline sounddevice.py:1648]: I don't think it is obvious to most readers what exactly "stable" means here?

**Comment by mgeier** [comment]
Thanks for this PR!

I think I would like this to be a larger change.

This sentence is copied from the PortAudio docs, but it is quite useless in the `sounddevice` context:

            Where practical, implementations should configure their
            latency based on this parameter, otherwise they may choose
            the closest viable latency instead.  Unless the suggested
            latency is greater than the absolute upper limit for the
            device, implementations should round the *latency* up to the
            next practical value -- i.e. to provide an equal or higher
            latency  wherever possible.

Isn't it?

Even if the information is useful, it's written from the wrong point of view.

Would you like to fix this?

Also, I don't like that there is divergent information between the `latency` argument to the `Stream` constructor and [sd.default.latency](https://python-sounddevice.readthedocs.io/en/0.3.15/api/module-defaults.html#sounddevice.default.latency).

I think I'd prefer to document only one and link to it from the other one.

That's also what I've done with [sd.default.blocksize](https://python-sounddevice.readthedocs.io/en/0.3.15/api/module-defaults.html#sounddevice.default.blocksize).

**Comment by mgeier** [comment]
> In my commit history there's a revert. Does this impact you in any way? Or do my commits just get merged into one at your end?

No problem, I can squash the commits in the end.

**Pull request merged**